src/core/plot/qgsplot.h

```diff
@@ -530,6 +530,13 @@ class CORE_EXPORT QgsPlotDefaultSettings
      */
     static QgsFillSymbol *chartBorderSymbol() SIP_FACTORY;
 
+    /**
+     * Returns the default line symbol to use for subsections lines.
+     *
+     * \since QGIS 3.44
+     */
+    static QgsLineSymbol *subSectionsSymbol() SIP_FACTORY;
```

Comment:
This doesn't belong here -- this class is generic and should not contain any elevation profile related code. I'd move it to QgsProfileRenderer instead and rename as "defaultSubSectionsSymbol".

Reply:
Moved to `QgsProfileRenderer` and renamed.